<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AnswerImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Обработка</string>
    </property>
    <addaction name="repair"/>
   </widget>
   <widget class="QMenu" name="PluginMenu">
    <property name="title">
     <string>Плагины</string>
    </property>
    <addaction name="refresh_plugins"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="PluginMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="refresh_plugins">
   <property name="text">
    <string>Обновить плагины</string>
   </property>
  </action>
  <action name="repair">
   <property name="text">
    <string>Отремонитровать</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
